Source/iOS/OIDAuthState+IOS.m

```diff
@@ -34,4 +34,12 @@ @implementation OIDAuthState (IOS)
                                                 callback:callback];
 }
 
++ (id<OIDExternalUserAgentSession>)
+    authStateByPresentingAuthorizationRequest:(OIDAuthorizationRequest *)authorizationRequest
+                                     callback:(OIDAuthStateAuthorizationCallback)callback {
+  return [self authStateByPresentingAuthorizationRequest:authorizationRequest
+                                presentingViewController:[UIViewController new]
```

Comment:
I don't like the use of `[UIViewController new]` as it bypasses the intent here.

Instead I think there should be a constructor for OIDExternalUserAgentIOS that doesn't require a UIViewController, and that is what should be used.